### PR TITLE
chore(release): release packages 🤘

### PR DESCRIPTION
**Pull request by gfellerph**

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

File: .changeset/pre.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "mode": "pre",
+  "tag": "beta",
+  "initialVersions": {
+    "@swisspost/design-system-changelog-github": "1.0.2",
+    "@swisspost/design-system-components": "1.0.6",
+    "@swisspost/design-system-components-angular": "1.0.6",
+    "@swisspost/design-system-intranet-header": "3.0.6",
+    "@swisspost/design-system-demo": "5.2.0",
+    "@swisspost/design-system-documentation": "1.2.0",
+    "@swisspost/internet-header": "1.4.2-beta.0",
```
Comment (alizedebray) [inline .changeset/pre.json:11]: Shouldn't it be `1.5.0-beta.1`?

Comment (gfellerph) [reply .changeset/pre.json:11]: This lists the state when the "changeset pre enter beta" command was executed, meaning before the changesets for the beta were applied with `changeset version`. I'm not sure why changeset needs this list, but it reads like: that's what the repo looked like before you entered beta mode (at the moment it's only possible to enter a pre mode for the whole monorepo and not individual packages, that makes stuff like releasing only one package as pre-release a little difficult).

**Review by alizedebray — APPROVED** [review]

**Comment by gfellerph** [comment]
Released the beta version manually

**Pull request closed without merging**